site/about/styles.css

```diff
@@ -647,71 +493,41 @@
 
      /* TOP PAGE */
 
-     .info-box-mobile {
-         display: flex;
-         flex-direction: row;
+     .info-sub-row {
          overflow: scroll;
      }
 
-     .mobile-box-info-cards {
-         padding: 100px 200px 100px 200px;
-         background-color: lightgray;
-         margin: 20px;
-         text-align: center;
-     }
-
-
      /* INFORMATION ROW ONTOP */
-     .campus-image-mobile-edit-1 {
-         width: 100%;
-         margin: 20px;
-     }
-
-     .campus-image-mobile-edit-2 {
-         width: 100%;
-         margin: 20px;
-     }
-
-     .campus-image-mobile-edit-3 {
-         width: 100%;
-         margin: 20px;
-     }
-
-     .campus-image-mobile {
-         overflow: scroll;
-         display: flex;
-         flex-direction: row;
+     .campus-images-alignment {
+         overflow: auto;
      }
 
      .campus-more-info {
          color: lightskyblue;
-         margin-left: 400px;
+         margin-top: 20px;
+         margin-left: 450px;
          text-decoration: underline;
      }
 
      /* CAMPUS ROW ONTOP */
-     .approach-mobile-card-row {
-         display: flex;
-         flex-direction: row;
+     .approach-card-row {
          overflow: scroll;
-         margin-left: 85px;
-         margin-top: 20px;
-         margin-right: 80px;
+         margin-left: 80px;
+         margin-right: 100px;
      }
 
-     .approach-1-mobile-info {
-         margin-bottom: 20px;
+     .approach-info-row {
+         flex-direction: column;
      }
 
-     .approach-mobile-card-edit {
-         margin-right: 95px;
-         border: 1px solid black;
+     .approach-1-info {
+         margin-right: 0px;
      }
 
      /* STUDENT WORK */
-     .student-work-mobile {
+     .student-work {
          position: absolute;
-         top: 4500px;
+         top: 5290px;
```

Comment:
Hmm, this seems iffy. You should probably make some parent element `position: relative` and then set the `top` on this. Right now I have a feeling that this `top` value is how far from the top of the page. Instead what you want is for it to be relative to the component it's in. We can chat about it in studio if you want.